sphinx-docs/Lateral-Movement-Guide.md

```diff
@@ -0,0 +1,92 @@
+not a domain account and (2) the user has administrative privileges and is a domain account are addressed in the 
+following sub sections.
+
+#### Non-Domain Account
+If the user with administrative privileges is *not* a domain account, remote UAC must be disabled on the target host by
+updating the following value in the registry. The command below will overwrite the existing registry value or add the
+vlaue if it doesn't already exist.
+```
+reg add HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System /v LocalAccountTokenFilterPolicy /t REG_DWORD /d 1 /f
+```
+After executing the above command, restart the host to apply the changes.
+
+
+#### Domain Account
+No additional setup should be necessary. 
+
+### Additional Considerations
+
+1. Ensure GPO/SRP or antivirus is not blocking remote access to shares.
+2. Ensure at least ADMIN$, C$, and IPC$ shares exist on the target host.
+
+## Lateral Movement Using CALDERA
+Lateral movement can be a combination of two steps. The first requires confirmation of remote access to the next target 
+host and the movement or upload of the remote access tool (RAT) executable to the host. The second part requires 
+*execution* of the binary, which upon callback of the RAT on the new host would complete the lateral movement. 
+
+Most of CALDERA's lateral movement and execution abilities found in Stockpile have fact or relationship requirements 
+that must be satisfied. This information may be passed to the operation in two ways:
+1. The fact and relationship information may be added to an operation's source. A new source can be created or this
+information can be added to an already existing source as long as that source is used by the operation. When configuring
+an operation, open the "**AUTONOMOUS**" drop down section and select "Use [insert source name] facts" to indicate to the 
+operation that it should take in fact and relationship information from the selected source.
+2. The fact and relationship information can be discovered by an operation. This requires additional abilities to be run
+prior to the lateral movement and execution abilities to collect the necessary fact and relationship information 
+necessary to satisfy the ability requirements. 
+
+### Moving the Binary
+There are several ways a binary can be moved or uploaded from one host to another. Some example methods used in 
+CALDERA's lateral movement abilities include:
+1. WinRM
+2. SCP
+3. wmic
+4. SMB
+5. psexec
+ 
+Based on the tool used, additional permissions may need to be changed in order for users to conduct these actions 
+remotely.
+
+### Execution of the Binary
+CALDERA's Stockpile execution abilities relevant to lateral movement mainly use wmic to remotely start the binary. Some 
+additional execution methods include modifications to Windows services, scheduled tasks, and Registry Run keys or start 
+up folder to execute the binary.
+
+## Example Lateral Movement Profile
```

Comment:
consider adding some screenshots of these options

Reply:
added the YML ability files at the end of the guide, also added a link to the Execution tactic of the ATT&CK page. Let me know if there's something specific you had in mind for the screenshots though, I'm not sure what they would be screenshots of.